src/swdmt/redistricting/LocationTest.java

```diff
@@ -1,11 +1,10 @@
 package swdmt.redistricting;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.number.OrderingComparison.greaterThan;
 import static org.hamcrest.number.OrderingComparison.lessThan;
-import org.junit.Test;
+import org.hamcrest.MatcherAssert;
```

Comment:
Since the only use associated with this import is for `assertThat`, use a static import, such as
```import static org.hamcrest.MatcherAssert.assertThat;```
Then clean up all of the instances of method invocations from `MatcherAssert.assertThat` to `asserThat`.